ui/components/app/connected-accounts-permissions/connected-accounts-permissions.component.js

```diff
@@ -64,20 +64,25 @@ export default class ConnectedAccountsPermissions extends PureComponent {
         >
           <p>{t('authorizedPermissions')}:</p>
           <ul className="connected-accounts-permissions__list">
-            {permissions.map(({ key: permissionName }) => (
-              <li
-                key={permissionName}
-                className="connected-accounts-permissions__list-item"
-              >
-                <CheckBox
-                  checked
-                  disabled
-                  id={permissionName}
-                  className="connected-accounts-permissions__checkbox"
-                />
-                <label htmlFor={permissionName}>{t(permissionName)}</label>
-              </li>
-            ))}
+            {permissions.map(({ key }) => {
+              const permissionName =
+                key === 'eth_accounts' ? 'permission_ethereumAccounts' : key;
+
+              return (
+                <li
+                  key={permissionName}
+                  className="connected-accounts-permissions__list-item"
+                >
+                  <CheckBox
+                    checked
+                    disabled
+                    id={permissionName}
+                    className="connected-accounts-permissions__checkbox"
+                  />
+                  <label htmlFor={permissionName}>{t(permissionName)}</label>
```

Comment:
Using a variable with the `t` function is discouraged, as it makes it difficult for our `verified-locales` lint script to determine whether or not the variable name is used. That script looks for string literals passed to `t` to ensure unused messages are detected and removed.

This is also probably why this mistake was introduced. Somebody searched for the old message key (`eth_accounts`) for any references to update, and this didn't show up in the search because that string isn't present here.

Reply:
That is good to know. Thank you! Working on creating a lookup table. First confirming we want to add flask support to this component...